libs/docs/platform/message-popover/examples/form-generator/form-generator-component-example.component.ts

```diff
@@ -24,6 +24,8 @@ export const dummyAwaitablePromise = (timeout = 200): Promise<boolean> =>
 @Component({
     selector: 'fdp-message-popover-form-generator-example',
     templateUrl: './form-generator-component-example.component.html',
+    changeDetection: ChangeDetectionStrategy.OnPush,
```

Comment:
Why do we need this? I think as a general rule of thumb, we are avoiding manual CD in the examples

Reply:
Yea, and this example does not uses changeDetectorRef ;)